src/include/stir/LORCoordinates.h

```diff
@@ -143,8 +143,9 @@ class LORCylindricalCoordinates_z_and_radius
     {
       check_state();
       assert(new_radius>0);
-      _z1 *= new_radius/_radius;
-      _z2 *= new_radius/_radius;
+      const auto mid_point = 0.5 * (_z1 + _z2);
+      _z1 = mid_point + (_z1 - mid_point) * new_radius / _radius;
+      _z2 = mid_point + (_z2 - mid_point) * new_radius / _radius;
```

Comment:
I guess you're saying that this fix isn't enough, as need to take angles into account?

If so, it is probably better to not adjust `_z1` and `_z2`, and adjust the documentation that `set_radius*` does *not* preserve the LOR.

Reply:
I think this fix is important nonetheless. Before, set_radius was also not correctly scaling up the LOR to a new radius and with this fix at least this behaviour is as expected. But I can definitely adjust the documentation to say that set_radius does not preserve the LOR.

Reply:
> with this fix at least this behaviour is as expected. 

well, I expected it to be the correct LOR, but it isn't. Not sure therefore what we achieve with this.

Reply:
Ok, I wasn't sure what the expected behaviour was meant to be :-) in this case I'll remove this fix from the PR.